core/client/example/go.mod

```diff
@@ -0,0 +1,50 @@
+module mykwilapp
+
+go 1.22.1
+
+require (
+	// this kuneiform rev uses the update sqlparser on kwil-db's main:
+	github.com/kwilteam/kuneiform v0.6.1-0.20240418203939-18f750d293bf
```

Comment:
When this commit goes on a release branch, this will change to the tagged version.  I had to use this unreleased revision of kuneiform for the `main` PR since there was an incompatible sqlparser change.